Functions/Quantification/xASL_quant_Basil.m

```diff
@@ -31,20 +34,28 @@
 % EXAMPLE: CBF_nocalib = xASL_quant_Basil(PWI, x);
 %
 % __________________________________
-% Copyright 2015-2021 ExploreASL 
+% Copyright 2015-2022 ExploreASL 
     
 
     %% Admin
     fprintf('%s\n','Quantification CBF using FSL BASIL:');    
     
     %% 1. Define paths
+    % For Basil and Fabber
     pathBasilInput = fullfile(x.dir.SESSIONDIR, 'PWI4D_BasilInput.nii');
     pathBasilOptions = fullfile(x.dir.SESSIONDIR, 'Basil_ModelOptions.txt');
-    dirBasilOutput = fullfile(x.dir.SESSIONDIR, 'BasilOutput');
+    dirBasilOutput = fullfile(x.dir.SESSIONDIR, 'Basil_Output');
+	
+	% Define if BASIL or Fabber is used - so that this is 
+	if isfield(x.modules.asl, 'bMultiTE') && x.modules.asl.bMultiTE == 1
+		bUseFabber = 1;
+	else
+		bUseFabber = 0;
+	end
         
-    %% 2. Delete previous BASIL output
-    xASL_adm_DeleteFileList(x.dir.SESSIONDIR, '(?i)^basilOutput.*$', 1, [0 Inf]);
-    FolderList = xASL_adm_GetFileList(x.dir.SESSIONDIR, '(?i)^basilOutput.*$', 'FPList', [0 Inf], 1);
+    %% 2. Delete previous BASIL/Fabber output
```

Comment:
Then please add it there ;)

This concerns a bit more. Let's say you first ran BASIL for getting a multi-PLD ATT and CBF map. Then you run it again to get the Texch map. In this case, you don't want to overwrite the first results with the other results. So at least the output should be saved somewhere. If we always ignore the output of BASIL/FABBER/whatever, then we might as well delete it when DELETETEMP is true, then we don't have this ambiguity.

Reply:
You can't run BASIL again to get Texch map - because it will always also recalculate the CBF and ATT map. Pointless not to overwrite. We should make it part of DELETETEMP. But it is good to overwrite here as well to avoid crashing.